sfn-query-large-datasets-cdk/app.py

```diff
@@ -0,0 +1,307 @@
+                        "glue:DeleteDatabase",
+                        "glue:CreateTable",
+                        "glue:UpdateTable",
+                        "glue:GetTable",
+                        "glue:GetTables",
+                        "glue:DeleteTable",
+                        "glue:BatchDeleteTable",
+                        "glue:BatchCreatePartition",
+                        "glue:CreatePartition",
+                        "glue:UpdatePartition",
+                        "glue:GetPartition",
+                        "glue:GetPartitions",
+                        "glue:BatchGetPartition",
+                        "glue:DeletePartition",
+                        "glue:BatchDeletePartition",
+                        "glue:StartCrawler",
+                        "glue:GetCrawler"
+                    ],
+                    resources=[
+                        f"arn:aws:glue:{self.region}:{self.account}:catalog",
+                        f"arn:aws:glue:{self.region}:{self.account}:database/{glue_db.database_input.name}",
+                        f"arn:aws:glue:{self.region}:{self.account}:table/{glue_db.database_input.name}/{glue_table}",
+                        f"arn:aws:glue:{self.region}:{self.account}:crawler/{glue_crawler.name}",
+                    ],
+                    effect=iam.Effect.ALLOW
+                )
+            ]
+        )
+
+        state_machine_role.attach_inline_policy(athena_workflow_policy)
+        state_machine_role.add_managed_policy( 
+            iam.ManagedPolicy.from_aws_managed_policy_name("CloudWatchLogsFullAccess"))
+        
+
+        # Step Functions State Machine
+        sfn_state_machine = sfn.CfnStateMachine(self, "sfn_athena",
+            state_machine_name=f"state-machine-start-athena-{random_string}",
+            role_arn=state_machine_role.role_arn,
+            definition_string=state_machine_definition,
+            definition_substitutions={
+                "aws_athena_workgroup": athena_workgroup.name,
+                "query_large_datasets_db": glue_db.database_input.name,
+                "topic_athena_updates": athena_updates_topic.topic_arn,
+                "glue_crawler_name": glue_crawler.name,
+                "glue_table": glue_table,
+            },
+            logging_configuration={
+                "level": "ALL",
+                "includeExecutionData": True,
+                "destinations": [{
+                    "cloudWatchLogsLogGroup": {
+                        "logGroupArn": log_group.log_group_arn
+                    }
+                }]
+            },
+            tracing_configuration={
+                "enabled": True
+            }
+        )      
+
+
+        # CloudFormation Outputs
+        outputs(self, "aws_sfn_state_machine", value=sfn_state_machine.ref)
+        outputs(self, "bucket_domain_name", value=s3_bucket.bucket_domain_name)
+        outputs(self, "aws_glue_crawler", value=glue_crawler.name)
```

Comment:
Why some outpouts are prefixed with `aws_` and others aren't? I'd remove that prefix.

Reply:
done.